modules/solid_mechanics/test/tests/dynamics/time_integration/direct_central_difference_multiVarBC.i

```diff
@@ -0,0 +1,128 @@
+###########################################################
+# This is a simple test with a time-dependent problem
+# demonstrating the use of a central difference with a
+# direct calculation of acceleration.
+#
+# Testing that the first and second time derivatives
+# are calculated correctly using the Central Difference Direct
+# method
+###########################################################
+
+[GlobalParams]
+    displacements = 'disp_x disp_y'
+[]
+
+[Mesh]
+    type = GeneratedMesh
+    dim = 2
+    xmin = -1
+    xmax = 1
+    ymin = -1
+    ymax = 1
+    nx = 4
+    ny = 4
+[]
+
+[Variables]
+    [disp_x]
+    []
+    [disp_y]
+    []
+[]
+
+[Functions]
+    [forcing_fn]
+        type = ParsedFunction
+        expression = 't'
+    []
+[]
+
+[Kernels]
+    [DynamicSolidMechanics]
+        displacements = 'disp_x disp_y'
+    []
+    [massmatrix]
+        type = MassMatrix
+        density = density
+        matrix_tags = 'system'
+        variable = disp_x
+    []
+    [massmatrix_y]
+        type = MassMatrix
+        density = density
+        matrix_tags = 'system'
+        variable = disp_y
+    []
+[]
+
+[Materials]
+    [elasticity_tensor_block_one]
+        type = ComputeIsotropicElasticityTensor
+        youngs_modulus = 1e1
+        poissons_ratio = 0.33
+    []
+
+    [strain_block]
+        type = ComputeFiniteStrain # ComputeIncrementalSmallStrain
+        displacements = 'disp_x disp_y'
+        implicit = false
+    []
+    [stress_block]
```

Comment:
Why doesn't the stress calculator require implicit=false?

Reply:
For consistency it should be, but I double checked and it doesn't change the solution when `implicit=false`.

Reply:
[50a87f9](https://github.com/idaholab/moose/pull/28431/commits/50a87f967cbbd8859f730a00c15bd65b8b557c1f)